python/tvm/relay/op/op.py

```diff
@@ -40,6 +40,40 @@ def get(op_name):
     return tvm.ir.Op.get(op_name)
 
 
+def register(op_name, describe=""):
```

Comment:
I think this is duplicated with ```register_op```, right?  Remove one and keep one you like.

Reply:
Yes, let's keep this one, since it is under relay.op namespace with other similar APIs.

Reply:
ok, let's keep this one